Shared/Samples/Apply raster rendering rule/ApplyRasterRenderingRuleView.swift

```diff
@@ -0,0 +1,113 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ApplyRasterRenderingRuleView: View {
+    /// A map with a streets basemap.
+    @State private var map = Map(basemapStyle: .arcGISStreets)
+    
+    /// An array of raster layers, each with a different rendering rule.
+    @State private var rasterLayers: [RasterLayer] = []
+    
+    /// The name of rendering rule selected by the picker.
+    @State private var selectedRenderingRule = "None"
+    
+    /// The viewpoint for zooming the map view to a layer's extent.
+    @State private var viewpoint: Viewpoint?
+    
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    var body: some View {
+        MapView(map: map, viewpoint: viewpoint)
+            .toolbar {
+                ToolbarItem(placement: .bottomBar) {
+                    Picker("Rendering Rule", selection: $selectedRenderingRule) {
```

Comment:
When the device is disconnected, the picker doesn't show up due to the rasterLayers is empty, but I think that is not a big deal for this sample.